samples/matter/common/src/bridge/ble_connectivity_manager.cpp

```diff
@@ -673,14 +683,27 @@ CHIP_ERROR BLEConnectivityManager::Reconnect(BLEBridgedDeviceProvider *provider)
 
 	StopScan();
 
-	bt_conn *conn;
-	bt_le_conn_param *connParams = GetScannedDeviceConnParams(provider->GetBLEBridgedDevice().mAddr);
+	bt_conn *conn{};
+	bt_addr_le_t btAddress = provider->GetBtAddress();
+	bt_le_conn_param *connParams = GetScannedDeviceConnParams(btAddress);
 
 	if (!connParams) {
 		LOG_ERR("Failed to get conn params");
 		return CHIP_ERROR_INTERNAL;
 	}
 
+#ifdef CONFIG_BRIDGE_FORCE_BT_CONNECTION_PARAMS
+	if (!CheckParamChangeRequest(connParams)) {
+		bt_le_conn_param forcedConnParams = {
+			.interval_min = CONFIG_BRIDGE_BT_CONNECTION_INTERVAL_MIN,
+			.interval_max = CONFIG_BRIDGE_BT_CONNECTION_INTERVAL_MAX,
+			.latency = CONFIG_BRIDGE_BT_CONNECTION_LATENCY,
+			.timeout = CONFIG_BRIDGE_BT_CONNECTION_TIMEOUT,
+		};
+		connParams = &forcedConnParams;
+	}
+#endif
```

Comment:
I could be wrapped in a function to avoid boiler plate (the same is used below).